spring-boot/proxy/src/main/java/eu/coatrack/proxy/security/exceptions/LocalApiKeyListWasNotInitializedException.java

```diff
@@ -0,0 +1,29 @@
+package eu.coatrack.proxy.security.exceptions;
+
+/*-
+ * #%L
+ * coatrack-proxy
+ * %%
+ * Copyright (C) 2013 - 2021 Corizon | Institut für angewandte Systemtechnik Bremen GmbH (ATB)
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import org.springframework.security.core.AuthenticationException;
+
+public class LocalApiKeyListWasNotInitializedException extends AuthenticationException {
```

Comment:
This name is not very clear. If I understand correctly, it is thrown when the gateway is started and directly goes to "offline mode", because the admin server is not reachable at the time of the gateway startup.

In other words: the gateway can not work in offline mode if it never was in online mode before. Maybe we should discuss the logic again. 

In any case, there should be JavaDoc added to explain the meaning of this exception.

Reply:
Added the JavaDoc.